src/i18n/strings/ar.json

```diff
@@ -676,7 +676,7 @@
     "Discovery": "الاكتشاف",
     "Deactivate account": "تعطيل الحساب",
     "Deactivate Account": "تعطيل الحساب",
-    "Deactivating your account is a permanent action - be careful!": "يعد تعطيل حسابك إجراءً دائمًا - كن حذرًا!",
+    "Deactivating your account is a permanent action —— ßbe careful!": "يعد تعطيل حسابك إجراءً دائمًا - كن حذرًا!",
```

Comment:
translation files not modified by `yarn i18n` cannot be touched as it causes merge conflicts in the translation system. We'd ultimately have to fix them in the translations system instead.

Reply:
Got it. Can you pls guide about how can this be done then

Reply:
You should only edit the `.tsx` file, delete all the other changes and then run `yarn i18n` which will generate changes to the English translation file. The other translation files are then going to be modified through https://translate.element.io/

Reply:
You should only edit the `.tsx` file, delete all the other changes and then run `yarn i18n` which will generate changes to the English translation file. The other translation files are then going to be modified through https://translate.element.io/

Reply:
I have reversed the changes in .json files. Following the procedure suggested by @SimonBrandner after editing the .tsx file, and running yarn i18n I still can't see any changes in the en_EN.json file. It still looks like this-
<img width="812" alt="Screenshot 2022-04-30 at 1 18 59 PM" src="https://user-images.githubusercontent.com/54431564/166096960-2b27518e-f0da-4c07-85bd-922d125eca25.png">
>

Reply:
I can still see a bunch of changes `JSON` files in the diff - did you push the changes? Are you saying that if you change the dash in the `.tsx` file and then run `yarn i18n` nothing the `en_EN.json` file doesn't get updated with the new dash?

Reply:
I have pushed the changes.  Still, I will check again.

Yes, actually I tried making the changes in a separate branch but nothing changes in the en_EN.json even after running yarn i18n.

Reply:
I have pushed all the changes. It can be viewed from the latest commit here
https://github.com/matrix-org/matrix-react-sdk/pull/8449/commits/20758ffd955291be39cd2d8ba27e1ef851a04439

Reply:
If you go to https://github.com/matrix-org/matrix-react-sdk/pull/8449/files, you can see there still are changes; it seems to be the case that the original code used `-` while you used `—`... I would suggest resetting to develop (something like `git reset --hard upstream/develop`), changing the `.tsx` file, running `yarn i18n` and then seeing what happens

Reply:
Just to make sure I am not doing anything wrong, I created a new branch did the change in .tsx file. Then ran yarn i18n but nothing is changed in en_EN.json